### ISSUE TITLE
I want to make new animation

### ISSUE THREAD

**Issue by nikhil8399** [facebookresearch/AnimatedDrawings#283]
I created a BVH file by rokoko, but now I have an issue with the retargeting file, so how do I make a retargeting file please help I am unable to understand the readme file content

**Comment by nikhil8399** [comment]
[2795750-uhd_3840_2160_25fps.zip](https://github.com/facebookresearch/AnimatedDrawings/files/15128381/2795750-uhd_3840_2160_25fps.zip)
This is my .bvh file

**Comment by hjessmith** [comment]
Have you tried searching the closed issues for a solution? In particular, the contents of #152 might be useful

**Comment by nikhil8399** [comment]
Yes, I tried this, but I do not understand this process. Please help

**Comment by nikhil8399** [comment]
I got this error when I made a retarget file. How do I solve this and find this error? Please help

![image_2024_04_29T11_57_14_323Z](https://github.com/facebookresearch/AnimatedDrawings/assets/142780734/39aed7ee-ffac-4da3-ba7d-c0c8f55ec5a7)

**Comment by nikhil8399** [comment]
This is the file in which I have a blank video, can you please check and help me, or can you please send the files for other animations?

[Uploading hosi.zip…]()
this is the black video that I created
![video](https://github.com/facebookresearch/AnimatedDrawings/assets/142780734/4dc6c655-a0bd-4548-a922-66d0b0db1ccd)
